packages/calcite-design-tokens/.prettierignore

```diff
@@ -0,0 +1,4 @@
+bin/
+dist/
+support/dist/
```

Comment:
Is a `dist` being created inside of `support/`? That seems like a strange design decision, can you explain why you chose that instead of something like `dist/support`?

Reply:
the idea is that the token-transformer will eventually be pulled out into it's own tool, completely separating responsibilities from the tokens themselves.